### PR TITLE
fix(icon ref): change from React.FC to forwardRef

### PR DESCRIPTION
**Pull request by doralchan**

### CODE REVIEW

File: src/sentry/static/sentry/app/icons/iconClose.tsx [hunk 1]
```diff
@@ -3,29 +3,41 @@ import React from 'react';
...
+export const IconClose = React.forwardRef(
+  (
+    {
+      color: providedColor = 'currentColor',
+      size: providedSize = 'sm',
+      circle: providedCircle = false,
+      ...props
+    }: IconProps,
+    ref: React.Ref<SVGSVGElement>
+  ) => {
```
Comment (billyvg) [inline src/sentry/static/sentry/app/icons/iconClose.tsx]: Looks like we're losing the component name in snapshots, I think we can get it back if we use a named function like below -- not sure how necessary this is though

```suggestion
export const IconClose = React.forwardRef(
  function IconClose(
    {
      color: providedColor = 'currentColor',
      size: providedSize = 'sm',
      circle: providedCircle = false,
      ...props
    }: IconProps,
    ref: React.Ref<SVGSVGElement>
  ) {
```

Comment (doralchan) [reply src/sentry/static/sentry/app/icons/iconClose.tsx]: Sure I'll add that

**Review by billyvg — APPROVED** [review]

File: tests/js/spec/components/__snapshots__/resolutionBox.spec.jsx.snap [hunk 2]
```diff
@@ -5,7 +5,7 @@ exports[`ResolutionBox render() handles default 1`] = `
   priority="default"
 >
   <BannerSummary>
-    <IconCheckmark
+    <ForwardRef(IconCheckmark)
```
Comment (doralchan) [inline tests/js/spec/components/__snapshots__/resolutionBox.spec.jsx.snap]: @billyvg Is this ok?

**Comment by billyvg** [comment]
Thanks for updating these!

**Pull request merged**